omeroserver/license/gpl-2.0.txt

```diff
@@ -0,0 +1,339 @@
+                 GNU GENERAL PUBLIC LICENSE
```

Comment:
:+1: for adding explicitly the license.

I understand that the choice `omeroserver/license/gpl-2.0.txt` comes from the `MANIFEST.in`. Is there a concrete rationale for doing so as opposed to putting `LICENSE.txt` in the top-level folder as per https://packaging.python.org/guides/distributing-packages-using-setuptools/#license-txt ?

Reply:
not really
There is a mix in our repo
``omero-web`` for example follows that approach, ``omero-py`` does not

Reply:
Understood. I suspect `omero-web` is actually the main exception in our Python projects to use such a license layout and that it comes from from the days where `OMERO.web` was a sub-component of https://github.com/ome/openmicroscopy with a different license.

Happy to hear otherwise from the @ome/python team, but my vote would be for switching to the recommended layout.